src/queries/types.ts

```diff
@@ -40,6 +40,7 @@ export const QUERY_TYPES = {
   EVENT_RECORDS: 'eventRecords',
   GENERIC_ITEM: 'genericItem',
   GENERIC_ITEMS: 'genericItems',
+  HOME_CONTENT_LIST: 'homeContentList',
```

Comment:
please find a different name without "HOME", as the list can be used everywhere, not just on home screens

Reply:
How about `staticContentList`?

Reply:
i like it even more than just `contentList`, yes, this shows directly that it comes from a static content config